deployment-bundle-steps.md

```diff
@@ -0,0 +1,39 @@
+# How to setup a project with deployment to S3
+
+This document assumes you have already setup the S3 bucket.
+
+## Using [CircleCi](https://circleci.com/)
+
+1. Log into [circleci.com](https://circleci.com/) and add your project. Set up the following credentials: (see other members of the team for these values).
+  - Environment Variables
+    - GHUSER
+    - GHPASS
```

Comment:
I think you have removed the reference to things that might want to use the password, so could safely remove this from this guide can't you? I might be misunderstanding but it looks to me like it says "this is our best practice, don't use two factor authentication here"